<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataGui</class>
 <widget class="QMainWindow" name="DataGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 205, 255);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);</string>
          </property>
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">border: 5px solid black;
font: 700 16pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Select Data</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="comboBox_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(193, 193, 193);</string>
            </property>
            <item>
             <property name="text">
              <string>Bar Graph</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pie Chart</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">border: 5px solid black;
font: 700 16pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Select Graph Type</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(206, 206, 206);</string>
            </property>
            <item>
             <property name="text">
              <string>Interest Based On Age</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Interest Based On Contact Type</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Interest Based On Organizer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Interest Based On Ethnicity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Interest Based On Education</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(206, 206, 206);</string>
            </property>
            <item>
             <property name="text">
              <string>Organizer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Recruitment_Attempts</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Contacts</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Point_History</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">border: 5px solid black;
font: 700 16pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>DOWNLOAD CSV</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);</string>
          </property>
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);</string>
          </property>
          <property name="text">
           <string>Clear Graphs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
